react/wbooks/src/app/components/Login/styles.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../../scss/colors';
+
+@import '../../../scss/components';
+
+.container {
+  border-top: 8px solid $cerulean;
+  margin: auto;
+  max-width: 300px;
+  padding: 24px;
+  width: 100%;
+}
+
+.error {
+  color: $red;
+}
+
+.errors-container {
+  margin-top: 15px;
+}
+
+.input {
+  border-radius: 5px;
+  height: 28px;
+}
+
+.input-label {
+  color: $black;
+  font-size: 15px;
+}
+
+.login-button {
+
+  @include button(); // stylelint-disable-line at-rule-no-unknown
```

Comment:
what is `button()` ? i think its in `scss/components` but it's a weird way to extend styles, that is why eslint is warning you. What about building a global `.button` class and use it in the JSX, and in the module you just declare a class that you can use for that specific button containing `background-color` and `color` attrs.

Reply:
I did it whit mixin because it was a Jota suggestion in the html and css training. I found it nice so I did it what way since is true that for example `login-button` *IS* a button, so why not "extend" it?
When do you use mixins in sass?

Reply:
Mixins is used when you are doing dynamic stuff, in this case you could use a extend/inheritance.

```
%button {
...button properties
}
```

and when you use a button that need to reuse those properties you just use `@extend %button`. This is because you are using static properties in that button, and that is why you have to call it like calling a function `()` weird, right?